emsymbolizer.py

```diff
@@ -102,10 +102,14 @@ def __init__(self):
     self.offsets = []
 
   def parse(self, filename):
-    with open(filename) as f:
-      source_map_json = json.loads(f.read())
-      if shared.DEBUG:
-        print(source_map_json)
+    try:
+      with open(filename) as f:
+        source_map_json = json.loads(f.read())
+        if shared.DEBUG:
+          print(source_map_json)
+    except Exception as e:
+      print('Could not open File/URL in sourceMappingURL')
+      raise e
```

Comment:
Can you check for the file existence before the try/catch and then error out without and exception in that case?

For json errors you can then be very specific and just wrap the `json.loads(f.read())` line and just catch `ValueError`.

Reply:
Yeah, I was unsure how much I wanted to do here. There's a try/catch that wraps the whole main function down at the bottom, and it catches this too with a decent error message (file not found, with the filename). This just makes it a little more clear why we tried to open that file. But that's a good point about loading the JSON: I had in mind file opening errors (which I ran into while testing this), but it might be good to do something different for JSON.
But this change is actually unrelated to this PR, so I should probably actually just back this out and make a separate PR.